config.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+// Config TODO
+type Config struct {
+	CustomerName string
+	Directory    map[string]interface{}
+	Slack        slackConfig
+	PagerDuty    pagerDutyConfig
+}
+
+// FromEnv TODO
+func (c *Config) FromEnv() error {
+	c.CustomerName = getVarFromEnv("CUSTOMER_NAME")
+
+	JSONDirectory := getVarFromEnv("SLACK_PAGERDUTY_DIRECTORY")
+	err := json.Unmarshal([]byte(JSONDirectory), &c.Directory)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = c.Slack.fromEnv()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
do not use Fatal in the `non-main` functions.

Reply:
Done.